internal/data/release_storage.go

```diff
@@ -0,0 +1,96 @@
+package data
+
+import (
+	"database/sql"
+	"time"
+
+	"github.com/ScienceSoft-Inc/integrity-sum/pkg/k8s"
+	"github.com/sirupsen/logrus"
+)
+
+//go:generate mockgen -source=release_storage.go -destination=mocks/mock_release_storage.go
+
+type IReleaseStorage interface {
+	PrepareQuery(deploymentData *k8s.DeploymentData) (string, []any)
+	Get(deploymentData *k8s.DeploymentData) (*Release, error) // TODO: remove, not use
+	Delete(nameDeployment string) error                       // TODO: remove, not use
+	DeleteOldData() error
+}
+
+type Release struct {
+	ID          int
+	Name        string
+	CreatedAt   time.Time
+	UpdatedAt   time.Time
+	ReleaseType string
+	Image       string
+}
+
+type ReleaseStorage struct {
+	db     *sql.DB
+	alg    string
+	logger *logrus.Logger
+}
+
+// NewReleaseStorage creates a new ReleaseStorage structure to work with the database table
+func NewReleaseStorage(db *sql.DB, alg string, logger *logrus.Logger) *ReleaseStorage {
+	return &ReleaseStorage{
+		db:     db,
+		alg:    alg,
+		logger: logger,
+	}
+}
+
+// PrepareQuery creates a query and a set of arguments for preparing data for insertion into the database
+func (rs ReleaseStorage) PrepareQuery(deploymentData *k8s.DeploymentData) (string, []any) {
+	args := make([]any, 0)
+
+	query := `INSERT INTO releases (name, created_at, updated_at, release_type, image) VALUES($1,$2,$3,$4,$5);`
+	args = append(args,
+		deploymentData.NameDeployment,
+		time.Now(),
+		time.Now(),
+		"type",
+		deploymentData.Image,
+	)
+	return query, args
+}
+
+// TODO: remove, not use
+// Get gets data from the database
+func (rs ReleaseStorage) Get(deploymentData *k8s.DeploymentData) (*Release, error) {
```

Comment:
and.. maybe remove it?

Reply:
removed https://github.com/ScienceSoft-Inc/integrity-sum/pull/48/commits/bd9d6c5ff362a3286b340bc959ce34e96eb90ab5